### PR TITLE
Package upgrade

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
Upgrade algo-builder version
Now we can use privateNet with all three wallets(i.e AlgoSigner, MyAlgo Wallet and Wallet Connect)

### CODE REVIEW

File: src/constants/constant.ts [hunk 1]
```diff
@@ -65,3 +65,7 @@ export const walletMessage = {
 	NETWORK_ISSUE: (walletType: string) =>
 		`Please select a network to connect with ${walletType}`,
 };
+
+export const mainnetURL = "https://node.algoexplorerapi.io";
```
Comment (vuvoth) [inline src/constants/constant.ts]: constant should have uppercase format.

File: src/types/interface.types.ts [hunk 2]
```diff
@@ -60,3 +60,18 @@ export interface DaoTableData {
 export interface UnknownObject {
 	[key: string]: string | number | boolean;
 }
+
+export interface AlgodTokenHeader {
```
Comment (vuvoth) [inline src/types/interface.types.ts]: Could you import those types from algo-builder?

**Review by vuvoth — COMMENTED** [review]
quick review!

File: src/constants/constant.ts (hunk 1 above)
Comment (Megha-Dev-19) [reply src/constants/constant.ts]: oh yes will update

File: src/types/interface.types.ts (hunk 2 above)
Comment (Megha-Dev-19) [reply src/types/interface.types.ts]: I will check if we export it from there, will update

File: src/components/WalletConnect.vue [hunk 3]
```diff
@@ -106,12 +106,19 @@ import {
...
+	MAIN_NET_URL,
+	BETA_NET_URL,
+	TEST_NET_URL,
+	walletMessage,
+} from "../constants/constant";
 import DaoID from "@/store/DaoID";
 import {
 	openErrorNotificationWithIcon,
 	WALLET_CONNECTION_ERROR,
 } from "@/constants";
```
Comment (vuvoth) [inline src/components/WalletConnect.vue:117]: What is different between `../constants/constant` and `@constants` ?

Comment (vuvoth) [inline src/components/WalletConnect.vue:119]: What is different between `../constants/constant` and `@constants` ?

**Review by vuvoth — APPROVED** [review]
Pre-approved! Thanks for your PR ~~

File: src/components/WalletConnect.vue (hunk 3 above)
Comment (rmeena840) [inline src/components/WalletConnect.vue]: You have tried to use another import from same folder. `"../constants/constant"` and `@/constants`. Please use import. I would recommend 2nd one.

**Review by rmeena840 — CHANGES_REQUESTED** [review]
Reviewed.

File: src/components/WalletConnect.vue (hunk 3 above)
Comment (Megha-Dev-19) [reply src/components/WalletConnect.vue:117]: `@constants` tells the folder from which to import i.e it's `index.tx` while `../constants/constant` is the specific address of the file

Comment (Megha-Dev-19) [reply src/components/WalletConnect.vue]: yeah you are right, I missed it, will update

**Review by rmeena840 — APPROVED** [review]
lgtm

**Pull request merged**